sphinx/templates/quickstart/make.ps1_t

```diff
@@ -0,0 +1,198 @@
+# 
+# .DESCRIPTION
+# This is a front-end script for Sphinx.
+# 
+# .EXAMPLE
+#  .\make.ps1 -clean
+# Cleanup the build directory.
+# .EXAMPLE
+#  .\make.ps1 -html
+# Build documentation as HTML pages.
+# .EXAMPLE
+#  .\make.ps1 -clean -html -linkcheck
+# Combine multiple commands in a single call
+# 
+# 
+# ==========================================================================
+# Copyright © 2016 Patrick Lehmann - Dresden, Germany
+# ==========================================================================
+[CmdletBinding()]
+param(
+  # Make all targets
+  [switch]$all =        $false,
+  # Make standalone HTML files
+  [switch]$html =       $false,
+  # Make HTML files named index.html in directories
+  [switch]$dirhtml =    $false,
+  # Make a single large HTML file
+  [switch]$singlehtml = $false,
+  # Make pickle files
+  [switch]$pickle =     $false,
+  # Make json files
+  [switch]$json =       $false,  
+  # Check all external links for integrity
+  [switch]$linkcheck =  $false,
+  # Clean up directory before running Sphinx.
+  [switch]$clean =      $false,
+  # Show the embedded help page(s).
+  [switch]$help =       $false
+)
+
+# resolve paths
+$WorkingDir =     Get-Location
+$SphinxRootDir =  Convert-Path (Resolve-Path ($PSScriptRoot))
+
+# set default values
+$EnableVerbose =  $PSCmdlet.MyInvocation.BoundParameters["Verbose"]
+$EnableDebug =    $PSCmdlet.MyInvocation.BoundParameters["Debug"]
+if ($EnableVerbose -eq $null)  { $EnableVerbose =  $false }
+if ($EnableDebug   -eq $null)  { $EnableDebug =    $false }
+if ($EnableDebug   -eq $true)  { $EnableVerbose =  $true  }
+
+# Display help if no command was selected
+$Help = $Help -or (-not ($all -or $html -or $dirhtml -or $singlehtml -or $pickle -or $linkcheck -or $clean -or $help))
+
+function Exit-Script
+{ <#
+    .PARAMETER ExitCode
+    ExitCode of this script run
+  #>
+  [CmdletBinding()]
+  param([int]$ExitCode = 0)
+  
+  # restore environment
+  # rm env:GHDL -ErrorAction SilentlyContinue
+  
+  cd $WorkingDir
+  
+  # unload modules
+  # Remove-Module precompile -Verbose:$false
+
+  # exit with exit code
+  exit $ExitCode
+}
+
+if ($Help)
+{ Get-Help $MYINVOCATION.InvocationName -Detailed
+  Exit-Script
+}
+
+if ($All)
+{ $clean =      $true
+  $html =       $true
+  $dirhtml =    $true
+  $singlehtml = $true
+}
+
+$SphinxBuild =    if (-not $env:SPHINXBUILD) { "sphinx-build" } else { $env:SPHINXBUILD }
+$BuildDir =       "$SphinxRootDir\{{ rbuilddir }}"
+$SourceDir =      "$SphinxRootDir\{{ rsrcdir }}"
+# $BuildDir =       "$SphinxRootDir\_build"   # for local testing, can be removed in the future
+# $SourceDir =      "$SphinxRootDir\."        # for local testing, can be removed in the future
+$AllSphinxOpts =  "-d $BuildDir\doctrees ${env:SPHINXOPTS} $SourceDir"
+$I18NSphinxOpts = "${env:SPHINXOPTS} $SourceDir"
+# ------------------------------------------------------------------------------
+# TODO: add paper options
```

Comment:
I have started to look into the `latex` target. The paper size option is needed for `latex`, so this was a hint for me that I need to translate this option in line 98.